starter-code/js/index.js

```diff
@@ -1,52 +1,76 @@
-function deleteItem(e){
-
-}
-
-function getPriceByProduct(itemNode){
-
-}
-
-function updatePriceByProduct(productPrice, index){
-
-}
-
-function getTotalPrice() {
-
-}
-
-function createQuantityInput(){
-
-}
-
-function createDeleteButton(){
-
-}
-
-function createQuantityNode(){
-
-}
-
-function createItemNode(dataType, itemData){
-
-}
-
-function createNewItemRow(itemName, itemUnitPrice){
-
-}
-
-function createNewItem(){
-
-}
+// window.onload = function(){
+//   var calculatePriceButton = document.getElementById('calc-prices-button');
+//   var createItemButton = document.getElementById('new-item-create');
+//   var deleteButtons = document.getElementsByClassName('btn-delete');
+
+//   calculatePriceButton.onclick = getTotalPrice;
+//   createItemButton.onclick = createNewItem;
+
+//   for(var i = 0; i<deleteButtons.length ; i++){
+//     deleteButtons[i].onclick = deleteItem;
+//   }
+// };
+
+var priceString = document.querySelectorAll(".price");
+var inputValue = document.querySelectorAll(".input");
+var displayTotalPrice = document.querySelectorAll(".total-price");
+var calculateBtn = document.querySelector(".btn-success");
+var deleteBtn = document.querySelectorAll(".btn-delete");
+var createBtn = document.querySelector(".btn-create");
+var createProduct = document.querySelector(".new-creation1");
+var createProduct1 = document.querySelector(".new-creation2");
+var addList = document.querySelector(".shopping-cart");
+
+calculateBtn.onclick = doCalculate;
+
+//calculate the prices + quantity
+function getTotalPrice(){
+   for ( i = 0; i < priceString.length; i++){
+    displayTotalPrice[i].innerHTML = (priceString[i].innerHTML) * (inputValue[i].value);
```

Comment:
innerHTML returns a string, so you should turn these values into numbers in order for this function to work :)
```js
function getTotalPrice() {
  for (i = 0; i < priceString.length; i++) {
    displayTotalPrice[i].innerHTML =
     parseInt( priceString[i].innerHTML) * parseInt(inputValue[i].value);
  }
}
```